python/nav/web/templates/seeddb/list_netbox.html

```diff
@@ -24,6 +25,8 @@
       {% else %}
         {% if element and forloop.counter0 in censor_list %}
```

Comment:
You still use censor_list here!

Reply:
Yes. I can remove it from the template too. Will do a cursory check to see whether the censor-list-code can be removed entirely from `render_list()` as well: I suspect the Netbox page was the only place it was used for anything.